src/index.html

```diff
@@ -8,20 +9,468 @@
     />
     <title>Catalog</title>
     <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
+      rel="stylesheet"
+      href="styles/index.scss"
     />
     <link
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
```

Comment:
The link to the Google Fonts API is duplicated. Consider removing one of these links to avoid unnecessary requests.